azure-iot-device/tests/iothub/aio/test_async_clients.py

```diff
@@ -949,6 +943,36 @@ def mock_open(self, mocker):
             client_class.create_from_edge_environment()
 
 
+@pytest.mark.describe("IoTHubModuleClient (Asynchronous) - .create_from_x509_certificate()")
+class TestIoTHubModuleClientCreateFromX509Certificate(IoTHubModuleClientTestsConfig):
+    @pytest.mark.it("Instantiates the client, given a valid X509 certificate object")
+    async def test_instantiates_client(self, client_class, x509):
+        client = client_class.create_from_x509_certificate(
+            hostname="durmstranginstitute.farend",
+            x509=x509,
+            device_id="MySnitch",
+            module_id="Charms",
+        )
+        assert isinstance(client, client_class)
+
+    @pytest.mark.it("Uses a X509AuthenticationProvider to create the client's IoTHub pipeline")
```

Comment:
>Uses a X5 [](start = 21, length = 9)

grammar nit: Even though X is a consonant, this is pronounced as "ex five oh nine", so it's more correct to say "Uses an X509AuthenticationProvider".